src/k-provider/ovp/config.js

```diff
@@ -2,7 +2,7 @@
 import {clone} from '../../util/clone'
 
 const defaultConfig: Object = {
-  serviceUrl: "//www.kaltura.com/api_v3",
+  serviceUrl: "//cdnapisec.kaltura.com",
```

Comment:
It needs to be https://cdnapisec.kaltura.com - need to check we don't add the protocol again in the code

Reply:
checked, we add the protocol to the cdnURL